### PR TITLE
mdx stanza: add test cases for paths

### PR DESCRIPTION
**Pull request by emillon**
This includes repro cases for #5596.

### CODE REVIEW

File: test/blackbox-tests/test-cases/mdx-stanza/paths.t [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  > ```ocaml file=/etc/passwd
+  > ```
+  > EOF
+
+  $ dune runtest
+  Internal error, please report upstream including the contents of _build/log.
+  Description:
+    ("Local.relative: received absolute path",
+    { t = "."; path = "/etc/passwd" })
+  Raised at Stdune__Code_error.raise in file "otherlibs/stdune/code_error.ml",
```
Comment (rgrinberg) [inline test/blackbox-tests/test-cases/mdx-stanza/paths.t:21]: Do you have a fix ready? Otherwise I suggest you redact the stacktrace. We don't want this test to be updated on unrelated commits.

Comment (emillon) [reply test/blackbox-tests/test-cases/mdx-stanza/paths.t:21]: I do but what's the technique to redact the stacktrace?

Comment (rgrinberg) [reply test/blackbox-tests/test-cases/mdx-stanza/paths.t:21]: Perhaps pipe it to awk?

```
.. | awk '/Internal error/,/Raised at/'
```

**Review by rgrinberg — APPROVED** [review]
